src/main/java/org/terasology/oreGeneration/generation/MappedStructureNodeTypeFieldFacet3D.java

```diff
@@ -25,11 +25,11 @@
 
     StructureNodeType get(CustomOreGen oreGenCreator, int x, int y, int z);
 
-    StructureNodeType get(CustomOreGen oreGenCreator, Vector3i pos);
+    StructureNodeType get(CustomOreGen oreGenCreator, Vector3ic pos);
```

Comment:
Let's add default methods here:
```suggestion
    StructureNodeType get(CustomOreGen oreGenCreator, Vector3ic pos) {
        return get(oreGenCreator, pos.x(), pos.y(), pos.z());
    }
```